packages/core/src/Dialog/Dialog.tsx

```diff
@@ -25,6 +25,7 @@ export interface IDialogProps {
   showCloseButton?: boolean
   size?: DialogSize
   triggerNode?: React.ReactNode
+  zIndex?: number | string
```

Comment:
It looks like we have a type defined for this https://github.com/priceline/design-system/blob/main/packages/core/src/theme/theme.ts#L271C13-L271C21

Reply:
Will update to that, thanks

Reply:
Updated